dev/cluster/server1.hcl

```diff
@@ -16,5 +16,5 @@ server {
   }
 
   # Self-elect, should be 3 or 5 for production
-  bootstrap_expect = 3
+  bootstrap_expect = 1
```

Comment:
This the dev cluster config, so bootstrap expect should match server2 and server3's.

You can override the value on the command line to use this config for a single server cluster or we could add a new config for single server clusters.